cp3-networkpolicy/egress/license-service-reporter/ibm-license-service-reporter-operator-egress.yaml

```diff
@@ -1,17 +1,15 @@
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
-  name: egress-ibm-license-service-reporter-operator
-  namespace: "lsrNamespace"
-  labels:
-    component: cpfs3
+  name: ibm-license-service-reporter-operator-kubernetes-egress
```

Comment:
why name changed? How to migrate from the earlier version? I.e. how to remove the earlier version from the cluster?

Reply:
name was change to such that telling more about proposes of network policy. in worst case will be duplicate of the same policy, because this one do not introduce nothing more.

Reply:
please revert to the previous filename, otherwise upgrades would end up with extranous old egress policy, ending up with ambiguity

Reply:
we have to find way to do such names migrations. cannot be like this that we will keep less readable names, because solution is not prepare to do changes. agree that can wait till working on Argo CD. now can put old name even if in worst case will be only duplicate of egress for customers which already have it on env. unfortunately new installed LSR will still have name egress name "egress-ibm-license-service-reporter-operator" which do not have any information for what is needed (like "ibm-license-service-reporter-operator-kubernetes-egress")